src/ui/DynamicHierarchicalFacetSearch/DynamicHierarchicalFacetSearchValue.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  IDynamicHierarchicalFacetSearchValueProperties,
+  IDynamicHierarchicalFacet
+} from '../DynamicHierarchicalFacet/IDynamicHierarchicalFacet';
+import { DynamicHierarchicalFacetSearchValueRenderer } from './DynamicHierarchicalFacetSearchValueRenderer';
+import { analyticsActionCauseList } from '../Analytics/AnalyticsActionListMeta';
+
+export class DynamicHierarchicalFacetSearchValue implements IDynamicHierarchicalFacetSearchValueProperties {
```

Comment:
This class is inspired by [`DynamicFacetValue`](https://github.com/coveo/search-ui/blob/master/src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts).